protocols/imap/src/test/java/org/apache/james/imap/processor/ListProcessorTest.java

```diff
@@ -143,12 +129,8 @@ public void convertMarkedUnknownChildrenShouldHaveMarkedFlagOnly() throws Except
             MailboxType.OTHER);
         assertThat(actual).isEqualTo(
             new ListResponse(
-                false,
-                false,
-                true,
-                false,
-                false,
-                false,
+                MailboxMetaData.Children.CHILDREN_ALLOWED_BUT_UNKNOWN,
```

Comment:
tests are not testing anything anymore

Reply:
Is that a bad thing? There is nothing more to test, (no more crazy wiring), and MPT is here for behavioural tests.

Reply:
Yes, having useless tests is a bad thing.
Making bad states not possible is a good thing and you did it but these tests need to go then.

Reply:
removed